conf/rewrite.conf

```diff
@@ -44,8 +44,17 @@
     Rule          $gn_invalid ^(true)$ from.rewrite.gn_invalid false
     Emitter_Name  gn_invalid
 
+[FILTER]
```

Comment:
This removes the injected metadata so the end result is the exact same log content